src/App.tsx

```diff
@@ -27,9 +27,25 @@ function App() {
           element={
             <PrivateRoute path="/">
               <Home />
-             </PrivateRoute>
+            </PrivateRoute>
           }
         >
+          <Route
+            path="/explore"
+            element={
+              <PrivateRoute>
+                <Explore />
+              </PrivateRoute>
```

Comment:
fix: I think explore page should not be private route it should be publicly accesible

Reply:
But firstly you need to login .